Shared/Managers/KeyboardManager.swift

```diff
@@ -124,7 +105,7 @@ class KeyboardManager {
 
     private func getInputAccessoryHeight() -> CGFloat {
         var inputAccessoryHeight: CGFloat = 0
-        if let responder = UIResponder.firstReponder,
+        if let responder = UIResponder.firstResponder,
```

Comment:
Whoops, "Responder" not "Reponder"

Reply:
You don't want to ponder on this responder?